plugin/src/main/kotlin/org/gradleweaver/plugins/jlink/JLinkPlugin.kt

```diff
@@ -10,21 +14,72 @@ open class JLinkPlugin : Plugin<Project> {
         const val EXTENSION_NAME = "jlink"
         const val JLINK_TASK_GROUP = "JLink"
         const val JLINK_TASK_NAME = "jlinkGenerate"
-        const val JLINK_ZIP_TASK_NAME = "jlinkZip"
+        const val JLINK_ARCHIVE_TASK_NAME = "jlinkArchive"
```

Comment:
If you want these to be statically accessible from the Groovy DSL or from a different java based plugin, these need to be annotated with `@JvmStatic`.

Reply:
That's not required for `const`-qualified fields according to IntelliJ:
> '@JvmStatic' annotation is useless for const or '@JvmField' properties